scripts/addMetamaskContractMapToHumanizerConsts.js

```diff
@@ -0,0 +1,35 @@
+const fsPromises = require('fs').promises
+const path = require('path')
+const humanizerInfo = require('../src/consts/humanizer/humanizerInfo.json')
+
+const humanizerInfoPath = path.join(
+  __dirname,
+  '..',
+  'src',
+  'consts',
+  'humanizer',
+  'humanizerInfo.json'
+)
+fetch('https://raw.githubusercontent.com/MetaMask/contract-metadata/master/contract-map.json')
+  .then((r) => r.json())
+  .then((mmContractMap) => {
+    Object.entries(mmContractMap).forEach(([address, info]) => {
+      const alreadyPresentData = humanizerInfo.knownAddresses[address.toLowerCase()]
+      humanizerInfo.knownAddresses[address.toLowerCase()] = {
+        ...alreadyPresentData,
+        address: address.toLowerCase(),
```

Comment:
Lowercasing the addresses is intentional, right? A while ago we discussed when to use checksummed addresses vs lower casing them.

Reply:
yes, it is intentional as the whole internally works with lowercase addresses. The migration should be in separate PR